test/unit/data/mockgen.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 var Swagmock = require('swagmock')
```

Comment:
Should these be `let` or `const`?

Reply:
Same, auto-generated by swaggerize. I have fixed it though.